crates/polars-mem-engine/src/executors/scan/ipc.rs

```diff
@@ -25,15 +25,17 @@ pub struct IpcExec {
 impl IpcExec {
     fn read(&mut self, verbose: bool) -> PolarsResult<DataFrame> {
         let is_cloud = self.paths.iter().any(is_cloud_url);
-        let mut out = if is_cloud || config::force_async() {
+        let force_async = config::force_async();
+
+        let mut out = if is_cloud || force_async {
             #[cfg(not(feature = "cloud"))]
             {
                 panic!("activate cloud feature")
             }
 
             #[cfg(feature = "cloud")]
             {
-                if !is_cloud && verbose {
+                if force_async && verbose {
```

Comment:
Always print forced async, as this PR makes `expand_paths` return paths in the format `file://...` which ends up being considered a cloud URL.